utils/node6-transform/index.js

```diff
@@ -19,18 +19,24 @@ const path = require('path');
 const removeRecursive = require('rimraf').sync;
 const transformAsyncFunctions = require('./TransformAsyncFunctions');
 
-const dirPath = path.join(__dirname, '..', '..', 'lib');
-const outPath = path.join(__dirname, '..', '..', 'node6');
-const fileNames = fs.readdirSync(dirPath);
-const filePaths = fileNames.filter(fileName => fileName.endsWith('.js'));
+copyFolder(path.join(__dirname, '..', '..', 'lib'), path.join(__dirname, '..', '..', 'node6'));
+copyFolder(path.join(__dirname, '..', '..', 'test'), path.join(__dirname, '..', '..', 'node6-test'));
 
-if (fs.existsSync(outPath))
-  removeRecursive(outPath);
-fs.mkdirSync(outPath);
 
-filePaths.forEach(filePath => {
-  const content = fs.readFileSync(path.join(dirPath, filePath), 'utf8');
-  const output = transformAsyncFunctions(content);
-  fs.writeFileSync(path.resolve(outPath, filePath), output);
-});
+function copyFolder(source, target, root) {
+  if (fs.existsSync(target))
+    removeRecursive(target);
+  fs.mkdirSync(target);
 
+  fs.readdirSync(source).forEach(file => {
+    const curSource = path.join(source, file);
+    if (fs.lstatSync(curSource).isDirectory()) {
+      copyFolder(curSource, path.join(target, path.basename(curSource)));
+    } else {
+      let text = fs.readFileSync(curSource);
+      if (curSource.endsWith('.js'))
+        text = transformAsyncFunctions(text.toString()).replace(/require\('\.\.\/lib\//g, `require('../node6/`);
```

Comment:
doesn't copying everything into a `node6/` work? this would spare you from this replace.

I.e.:
```js
removeSync('node6');
copyFolder('lib', 'node6/lib');
copyFolder('test', 'node6/test');
copyFile('index.js', 'node6/index.js');
```

Reply:
It gets really tricky because we load some utils from lib, and some utils are hard to convert. There are also paths for things like `package.json` and `.local-chromium` which get confusing.